include/odp/api/spec/ipsec.h

```diff
@@ -192,6 +234,33 @@ typedef struct odp_ipsec_outbound_config_t {
 
 } odp_ipsec_outbound_config_t;
 
+/**
+ * IPSEC TEST capability
+ */
+typedef struct odp_ipsec_test_capability_t {
+	/** Support for updating internal parameters of the SA for test */
+	odp_support_t sa_update;
```

Comment:
This looks redundant. It would seem to me that the individual parameter specific capabilities would be sufficient. If none of them is set, then update is not supported at all.

Reply:
Agreed. Removed odp_support_t sa_update from capability with v5.